WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1796,7 +1796,9 @@ private void saveResult(boolean saved, boolean uploadNotStarted) {
     }
 
     private void showPrepublishingNudgeBottomSheet() {
-        PrepublishingBottomSheetFragment prepublishingFragment = PrepublishingBottomSheetFragment.newInstance();
+        ActivityUtils.hideKeyboard(this);
+        PrepublishingBottomSheetFragment prepublishingFragment =
+                PrepublishingBottomSheetFragment.newInstance(getSite());
```

Comment:
Usually it's preferred to share an id instead of sharing the whole model. The max size of the bundle is limited and we could end up with a TransactionTooBigException. In case it's wired in the re-used fragment, don't worry about it.

Reply:
Yes, it's wired in the re-used fragment sadly. Our codebase suffers from this in a lot of places. We will get to them eventually 😄

Reply:
`PrepublishingBottomSheetFragment` is a new fragment and inherits only from the base `Fragment`, right? I understand we can't easily fix it in the `PrepublishingTagsFragment` but it seems this(PrepublishingBottomSheetFragment) could be easily fixed. Wdyt?

Reply:
Oh, the `SiteModel` is being passed here so that it can be further passed to the `TagsFragment` that needs the entire model. Let me know if this clears it up or if there's a way to do this more efficiently.

Reply:
I see. Ideally we wouldn't introduce non-ideal patterns in new fragments. One solution would be to pass only the siteId into the `PrepublishingBottomSheetFragment` -> load the siteModel from the database -> and pass it to the legacy fragment. However, I'm not sure it's worth it at the moment.

Reply:
Thanks, yes, this was something I was thinking of doing but wasn't sure it was worth it either so I didn't but if from an architecture standpoint it should be done I could definitely do the change.